frontend/public/components/dashboards-page/overview-dashboard/overview-dashboard.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+import { Dashboard, DashboardGrid } from '../../dashboard';
+import { HealthCard } from './health-card';
+
+export const OverviewDashboard: React.FC<{}> = () => {
+  const mainCards = [
+    <HealthCard key="health" />,
+  ];
+
+  return (
+    <Dashboard>
+      <DashboardGrid mainCards={mainCards} />
```

Comment:
Shouldn't `DashboardGrid`'s `mainCards` prop be mandatory? (with `leftCards` and `rightCards` remaining to be optional)

In other words, the "main" column of `DashboardGrid` is always expected to have some content, right?

Reply:
probably yes. Marked `mainCards` as required